pnpm-lock.yaml

```diff
@@ -71,6 +71,9 @@ importers:
       react-hot-toast:
         specifier: ^2.4.1
         version: 2.4.1(csstype@3.1.3)(react-dom@18.2.0(react@18.2.0))(react@18.2.0)
+      react-icons:
+        specifier: ^5.3.0
+        version: 5.3.0(react@18.2.0)
```

Comment:
Please reverse this, your `pnpm` file should not be updated.

Reply:
I don't recall updating my pnpm. Is this not showing a version for react-icons?